### PR TITLE
Closes #1331: Make sidebar not overlap text for Text with Background

### LINKED ISSUE #1331: Sidebar menu covers up content on Text with Background
### Describe the bug
While working on the new Think Tank website, I noticed that the Sidebar menu (added by default to the Sidebar-first region) covers up the content on Text with Background.

### To Reproduce
1. Create a Drupal page and add Text on Media
2. Go to Structure and select Block Layout
3. Place the Sidebar menu under the Sidebar First region
4. Ensure the menu shows up on your new page

### Screenshots of the issue
Screen Shot 2022-02-18 at 2 39 33 PM

### Expected behavior
The sidebar menu should push the content over so it doesn't cover it up.

### PR DESCRIPTION
**Pull request by smayden**
## Description
Simple fix of the class specified in the modules/custom/az_paragraphs/css/az_paragraphs_full_width.css to match the class on the elements in Text with Background

## Related Issue
#1331 

## How Has This Been Tested?
Locally and in Probo
Appreciate testing to make sure it doesn't break Text on Media (particularly when using Media Aspect Ratio) or other full-width elements.

https://5ec40ba7-e988-4dc9-9e2f-0560bea3e812.probo.build/pages/test-page

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by akslay** [comment]
I rearranged the elements on this page and pulled one up that looked like it might overlap: https://5ec40ba7-e988-4dc9-9e2f-0560bea3e812.probo.build/pages/text-media
And it does overlap. But I'm not sure if that's a symptom of this PR or if this was already happening prior to this PR. Text on Background looks good though.

**Comment by smayden** [comment]
I would guess a symptom of this PR... Weird that it is only happening for me with that particular Text on Media paragraph. I can't recreate it with a different video and the same text/settings.

**Comment by akslay** [comment]
Yeah that's the only one I see it happening on too, which is odd.

**Comment by smayden** [comment]
Actually, maybe this is not related to this PR and is yet another exciting sidebar bug. I am able to recreate the issue on a different Probo build: https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1353.probo.build/pages/text-media

**Review by akslay — APPROVED** [review]

**Review by danahertzberg — APPROVED** [review]

**Review by joeparsons — DISMISSED** [review]
Code changes look good to me.

File: modules/custom/az_paragraphs/css/az_paragraphs_full_width.css [hunk 1]
```diff
@@ -107,14 +107,11 @@
...
         -ms-flex: 0 0 75%;
         flex: 0 0 75%;
         max-width: 75%;
-    }
-    .layout-sidebar-first .region-content .full-width-background .az-full-width-row {
-        margin-right: -30px;
-        margin-left: -30px;
+        margin-left: auto;
         -ms-flex-pack: end;
         justify-content: flex-end;
```
Comment (danahertzberg) [inline modules/custom/az_paragraphs/css/az_paragraphs_full_width.css:116]:
```suggestion
```

File: modules/custom/az_paragraphs/css/az_paragraphs_full_width.css [hunk 2]
```diff
@@ -157,6 +152,10 @@
         flex: 0 0 50%;
         max-width: 50%;
     }
+    .layout-sidebar-first .region-content .az-full-width-column-content {
```
Comment (danahertzberg) [inline modules/custom/az_paragraphs/css/az_paragraphs_full_width.css:155]:
```suggestion
    .layout-sidebar-first .region-content .full-width-background .az-full-width-column-content {
```

**Comment by smayden** [comment]
I _think_ this is now ready for review:
https://7b5e2985-cedf-4a13-ab91-3a35916c835d--pr-1360.probo.build/

**Review by danahertzberg — APPROVED** [review]

**Review by trackleft — APPROVED** [review]

**Pull request merged**